packages/logo/src/Logo.php

```diff
@@ -35,13 +35,19 @@ public function __construct( $url = '' ) {
 	/**
 	 * Build and render an <img /> tag with the Jetpack logo.
 	 *
+	 * @param string $type If empty, the color version is used. If it's 'gray', a gray version of the logo will be displayed.
+	 *                     The gray it's actually the blueish gray version used in Calypso's colophon found in Stats.
+	 *
 	 * @return string The Jetpack logo in an <img /> tag.
 	 */
-	public function render() {
+	public function render( $type = '' ) {
```

Comment:
It might be worth making an assertion here for now that valid values are 'grey' and ''.